projects/plugins/jetpack/modules/contact-form.php

```diff
@@ -19,7 +19,8 @@
  * Feature: Writing
  * Additional Search Queries: contact, form, grunion, feedback, submission, contact form, email, feedback, contact form plugin, custom form, custom form plugin, form builder, forms, form maker, survey, contact by jetpack, contact us, forms free
  */
-
+add_filter( 'jetpack_contact_form_use_package', '__return_true' );
```

Comment:
I don't think we want these changes here :)

Reply:
Indeed, this was a way to facilitate testing. I'll remove it.